platform/podjs/package.json

```diff
@@ -13,7 +13,8 @@
     "build": "shx rm -rf tsconfig.tsbuildinfo dist && tsc --emitDeclarationOnly && rollup -c",
     "build-downstream": "../../build.js --start @polypoly-eu/podjs",
     "lint": "eslint src --ext .ts",
-    "test": "jest && cypress run"
+    "test": "jest",
```

Comment:
Any reason why you're eliminating the `cypress` test?

Reply:
hmm smells wrong conflict resolution